cityofaustin/atd-moped
# Move map from Map tab to Project list view

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/16966

This one moves the projects map from a tab in the main app view to the project list view where it is toggled on/off with a switch. I'd appreciate any feedback on design, etc. 🙏 No rush since the [next issue](https://github.com/cityofaustin/atd-data-tech/issues/9927) isn't blocked by this.

## Testing
**URL to test:** 

Local

**Steps to test:**
1. Go to the Projects table
2. See that there is a switch next to the **Download** export button that toggles the map view
3. See that the map that was in a tab in the [last PR](https://github.com/cityofaustin/atd-moped/pull/1276) now replaces the list when the switch is on
4. I also made changes to the responsive behavior of the row with the search bar, download button, and switch so feedback on that would be great, thanks!

---
#### Ship list
